utils/diagram_generator/lib/main.dart

```diff
@@ -62,6 +63,7 @@ Future<Null> main() async {
     CustomListItemDiagramStep(controller),
     FlowDiagramStep(controller),
     HeroesDiagramStep(controller),
+    IconDiagramStep(controller),
```

Comment:
Sorry about some of this formatting / import noise -- this is the only change I actually made. The rest is `dartfmt` doing its thing.